src/main/webapp/scripts/fetchData.js

```diff
@@ -17,8 +18,8 @@ function fetchSpeciesData(name) {
         var genusContainer          = document.getElementById('genus-container');
 
         // Update names for species
-        commonNameContainer.innerText       = speciesData.commonName;
-        scientificNameContainer.innerText   = speciesData.binomialName;
+        commonNameContainer.innerText       = capitalizeSpeciesName(speciesData.commonName);
```

Comment:
Why are you capitalizing the name here?

Reply:
There was a TODO to capitalize the names, so I added it here as well

Reply:
oh hmm. If it's just for design purposes, it might be better to keep it as written here and then capitalize it with the css

Reply:
Sure, I can revert it and add the change using CSS Instead